web/cobrands/fixmystreet/staff.js

```diff
@@ -456,7 +456,7 @@ $(fixmystreet).on('report_new:category_change', function(evt, $this) {
     var role = $this.data('role');
```

Comment:
Can remove this now (and the setting of data-role in the HTML, given this was only thing using it).